lib/mayaUsd/fileio/primUpdaterManager.h

```diff
@@ -78,6 +80,53 @@ class PrimUpdaterManager : public PXR_NS::TfWeakBase
     bool hasPulledPrims() const { return _hasPulledPrims; }
 
 private:
+    struct VariantSelection
```

Comment:
I think there are/were other stories related to doing things when pulled objects become stale?

What I''m getting at is that we may have to record more data related to pulled objects in the future and maybe these data types and their management would be better off in a separate class that the prim updater manager would hold an instance of instead of having all the types and code directly in it. If we do end-up adding more things to manage, putting it all directly in the prim updater manager will become unwieldy. Only those variants already add quite a bit of data that is not really related to prim updates.

We may wait until we get to these other stories/issues, but it would add a burden of refactoring to these stories.